app/views/find/search/visa_status/new.html.erb

```diff
@@ -21,9 +21,18 @@
         exclude_keys: ["visa_status"]
       ) %>
 
-    <%= f.govuk_radio_buttons_fieldset :visa_status, legend: { text: I18n.t("find.visa_status.title"), size: "l", tag: "h1" } do %>
+    <%= f.govuk_radio_buttons_fieldset :visa_status, legend: { text: I18n.t("find.visa_status.primary_title"), size: "l", tag: "h1" } do %>
+
+    <p class="govuk-body">You’ll need to have the right to work or study in the UK for the duration of your teacher training course.</p>
+
+    <p class="govuk-body"><%= govuk_link_to("Check if you need a UK visa and find out how to apply", t("find.get_into_teaching.url_visas_for_non_uk_teacher")) %>.</p>
+
+    <p class="govuk-body">If you do not already have the right to work or study, you should only apply to courses that have visa sponsorship available.</p>
+
+    <h2 class="govuk-heading-l"><%= t("find.visa_status.secondary_title") %></h2>
```

Comment:
@MaeveRoseveare should the h2 class here be "govuk-heading-m"?

I think the fact that the H1 also uses "govuk-heading-l" means they end up being the same size?

![image](https://github.com/DFE-Digital/publish-teacher-training/assets/35870975/a17de858-94e7-47ae-86e0-4684b02a4fcd)